apps/asap-server/src/queries/research-outputs.queries.ts

```diff
@@ -0,0 +1,276 @@
+import gql from 'graphql-tag';
+
+export const FETCH_RESEARCH_OUTPUT = gql`
+  query FetchResearchOutput($id: String!, $withTeams: Boolean!) {
+    findResearchOutputsContent(id: $id) {
+      id
+      created
+      lastModified
+      flatData {
```

Comment:
i wasn't able to either make use of the fragments or embed a string inside the query string - seems like apollo is pretty stupid about reading those queries as it can only read whats between the `gql` tag and generate the type for that

Reply:
its done